app/code/Magento/Bundle/Pricing/Price/ConfiguredPrice.php

```diff
@@ -146,10 +152,9 @@ public function getValue()
     {
         if ($this->item) {
             $configuredOptionsAmount = $this->getConfiguredAmount()->getBaseAmount();
-            return parent::getValue() +
-                $this->priceInfo
-                    ->getPrice(BundleDiscountPrice::PRICE_CODE)
-                    ->calculateDiscount($configuredOptionsAmount);
+            if (!empty($this->item->getProduct())) {
```

Comment:
I did it in this way because $this->item can be the instance of null|ItemInterface. And this interface also implements another models. And as this branch of code is fallback. I did not reserch so deep but i think that "if" statement need to  check only $this->item. Maybe i wrong.
@bgorski